test/MicrosoftTeams.spec.ts

```diff
@@ -1972,6 +1972,83 @@ describe("MicrosoftTeams", () => {
     });
   });
 
+  describe("sendBotRequest", () => {
+    it("should not allow calls before initialization", () => {
+      expect(() =>
+      microsoftTeams.sendBotRequest({ query: "" }, () => {
+        return;
+      })
+    ).toThrowError("The library has not yet been initialized");
+    });
+    it("should successfully send a request", () => {
+      initializeWithContext("content");
+      const request = {
+        query: "some query"
+      };
+
+      let botResponse: BotResponse;
+      let error: string;
+
+      const handleBotResponse = (response: BotResponse) => (botResponse = response);
+      const handleError = (_error: string): any => (error = _error);
+
+      // send message request
+      microsoftTeams.sendBotRequest(request, handleBotResponse, handleError);
+
+      // find message request in jest
+      const message = findMessageByFunc("executeBotQuery");
+
+      // check message is sending correct data
+      expect(message).not.toBeUndefined();
+      expect(message.args).toContain(request);
+
+      const _botResponse: BotResponse = { data: ["some", "queried", "items"] };
+
+      // simulate response
+      const data = {
+        success: true,
+        response: _botResponse
+      };
+
+      respondToMessage(message, data.success, data.response);
+
+      // check data is returned properly
+      expect(botResponse).toBe(_botResponse);
+      expect(error).toBeUndefined();
+    });
+    it("should invoke error callback", () => {
+      initializeWithContext("content");
+      const request = {
+        query: "some broken query"
+      };
+
+      let botResponse: BotResponse;
+      let error: string;
+
+      const handleBotResponse = (response: BotResponse) => (botResponse = response);
+      const handleError = (_error: string): any => (error = _error);
+
+      microsoftTeams.sendBotRequest(request, handleBotResponse, handleError);
+      const message = findMessageByFunc("executeBotQuery");
+      expect(message).not.toBeUndefined();
+      expect(message.args).toContain(request);
+
+      const _botResponse: string = "Something went wrong...";
```

Comment:
NIT: The `_botResponse` variable is not necessary here; consider defining it directly inline within the `data` object below.